src/general/GameWorld.cs

```diff
@@ -71,12 +71,26 @@ public class GameWorld : ISaveLoadable
     /// </summary>
     /// <param name="settings">Settings to generate the world with</param>
     /// <param name="startingSpecies">Starting species for the player</param>
-    public GameWorld(WorldGenerationSettings settings, Species? startingSpecies = null) : this()
+    public GameWorld(WorldGenerationSettings settings, Species? startingSpecies = null) : this(settings)
     {
-        WorldSettings = settings;
```

Comment:
I don't understand what happens to world settings applying here but I know you know what you're doing so I'll just ignore that.

Reply:
The world settings is now set by the other constructor which is made to run before this constructor starts (by this bit ` : this(settings)`) of the code